irods_capability_automated_ingest/irods_sync.py

```diff
@@ -199,9 +199,9 @@ def main():
                               type=str, default=None, help='URL to proxy for S3 access')
     parser_start.add_argument('--s3_insecure_connection', action="store_true",
                               default=False, help='Do not use SSL when connecting to S3 endpoint')
-    parser_start.add_argument('--s3_multipart_chunksize_in_mb',action="store", type=int,
+    parser_start.add_argument('--s3_multipart_chunksize_in_mib',action="store", type=int,
                               default=8, choices=range(5, 5001), metavar="[5-5000]",
-                              help='File chunk size in megabytes for files uploaded multi-part to Amazon S3 (MB). Minimum part size specified by AWS is 5 MB, and the maximum part size is 5000 MB.')
+                              help='File chunk size in mebibytes for files uploaded multi-part to S3. Minimum part size is 5 MiB, and the maximum part size is 5000 MiB.')
```

Comment:
What do you think of this rewording?
```suggestion
                              help='Chunk size in mebibytes for multipart S3 uploads. Minimum part size is 5 MiB and the maximum part size is 5000 MiB.')
```
I think the "uploaded multi-part" wording is tripping me up, so any kind of rewording there would seem better to me.

Reply:
Yeah, let's go with that.